README.md

```diff
@@ -93,7 +93,7 @@ __Server side__:
 When you sign the token with an expiration time:
 
 ```javascript
-var token = jwt.sign(user_profile, jwt_secret, {expiresInMinutes: 60});
+var token = jwt.sign(user_profile, jwt_secret, { expiresIn: 60 });
```

Comment:
There is no place in MMS that signs JWTs, right? MMS only does verification, correct?

Reply:
Correct. But I didn't want to leave the docs out of date since I updated the tests.